src/Symfony/Component/BrowserKit/AbstractBrowser.php

```diff
@@ -515,13 +507,9 @@ protected function filterResponse($response)
      *
      * This method returns null if the DomCrawler component is not available.
      *
-     * @param string $uri     A URI
-     * @param string $content Content for the crawler to use
-     * @param string $type    Content type
-     *
      * @return Crawler|null
      */
-    protected function createCrawlerFromContent($uri, $content, $type)
+    protected function createCrawlerFromContent(string $uri, string $content, ?string $type)
```

Comment:
`$type` is a string. `null` is allowed in `addContent()` just as a default value.